io-engine/src/core/env.rs

```diff
@@ -391,7 +430,7 @@ impl Default for MayastorEnvironment {
             bdev_io_ctx_pool_size: 65535,
             nvme_ctl_io_ctx_pool_size: 65535,
             nvmf_tgt_interface: None,
-            nvmf_tgt_crdt: 0,
+            nvmf_tgt_crdt: [0; 3],
```

Comment:
For the sake of readability, can we #define the value 3 to a suitable macro name.